templates/zerver/config_error/remote_billing_bouncer_not_configured.html

```diff
@@ -0,0 +1,9 @@
+{% extends "zerver/config_error/container.html" %}
+
+{% block error_content %}
+    {% trans %}
+        This server is not configured to use push notifications. For instructions on how to
+        configure push notifications, please see the
+        <a href="https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html">documentation</a>.
```

Comment:
Let's pull the URL out of the translation tag as a variable, like we do in various email templates.

Can you post a sample screenshot in chat.zulip.org so we can debate the exact language?

Reply:
Started a thread in czo. Is this a good way to get the url out of the translation tag?
```
    {% with doc_url="https://zulip.readthedocs.io/en/latest/production/mobile-push-notifications.html" %}
    {% trans %}
        This server is not configured to use push notifications. For instructions on how to
        configure push notifications, please see the
        <a href="{{ doc_url }}">documentation</a>.
    {% endtrans %}
    {% endwith %}
```

Reply:
I only know how to do it in the `trans` entry a la `templates/zerver/emails/remote_realm_billing_confirm_login.html` has an example

Reply:
Oh yeah, that's more concise, I can get rid of the with `with` tag